pkgs/top-level/all-packages.nix

```
@@ -7891,7 +7891,16 @@ with pkgs;
 
   grails = callPackage ../development/web/grails { jdk = null; };
 
-  graylog = callPackage ../tools/misc/graylog { };
+  graylog =  hiPrio graylog-3_3;
```

```suggestion
  graylog = graylog-3_3;
```
aliases normally have no prio attached to them

---

But in this case it would break installations of 3.3 users if we directly would go to version 5: https://docs.graylog.org/docs/upgrade-path

---

solved via release note